src/client/templates/export.html

```diff
@@ -1,8 +1,20 @@
 <template name="exportRoom">
     <body>
         <button class="btn btn-info" id="copyToClipboardButton" type="button">Copy To Clipboard</button>
-
+        {{#if getSummaryView}}
+        	<button class="btn btn-info" id="exportAllButton" type="button">Comprehensive View</button>
+        {{else}}
+        	<button class="btn btn-info" id="exportSummaryButton" type="button">Summery View</button>
+       	{{/if}}
        <textarea id="textArea" rows="50" cols="200" style="margin: 10px 10px 10px 10px" readonly>
+     	{{#if getSummaryView}}
+{{#each getCategories}}
+{{this.category}}
+{{#each getCards this.category}}
+Thought: {{text}}
+{{/each}}
+{{/each}}
+     	{{else}}
```

Comment:
indentation sucks here because it translates into tabs in the text area .  Does anyone know if there is a good way to fix this?